Tools/cocoapods.rb

```diff
@@ -20,7 +20,7 @@
 end
 
 # Common Settings
-SWIFT_VERSION = '4.2'
+SWIFT_VERSION = '5.0'
```

Comment:
We should update the README.md with the new Swift (and Xcode?) requirements as well.

Reply:
We also probably want to keep a Swift 4.2 branch and `Package@swift-4.2.swift`. I'll push a new commit.

Reply:
There were no source changes necessary for migrating to Swift 5, so I think having a `Package@swift-4.2.swift` should suffice.